tests/ship_log.cpp

```diff
@@ -265,12 +265,6 @@ BOOST_DATA_TEST_CASE(basic_prune_test, bdata::xrange(2) * bdata::xrange(2) * bda
       return e.to_detail_string().find("missed a fork change") != std::string::npos;
    });
 
-   //start from genesis not allowed
-   BOOST_REQUIRE_EXCEPTION(t.add(2, payload_size, 'A', 'A');, eosio::chain::plugin_exception, [](const eosio::chain::plugin_exception& e) {
-      std::string err = e.to_detail_string();
-      return err.find("existing ship log") != std::string::npos && err.find("when starting from genesis block") != std::string::npos;
-   });
-
```

Comment:
I think it's okay to remove this particular check: the test case is actually for pruned logs and pruned logs for this test case won't have the genesis block anyways. The test worked previously because the `blocknum==2` check was so early in the logic.

The test case further down exercises the genesis block behavior.